Makefile

```
@@ -69,6 +69,9 @@ STORAGE := $(or ${STORAGE},s3)
 GENERATE_CRD := $(or ${GENERATE_CRD},true)
 PERSISTENT_STORAGE := $(or ${PERSISTENT_STORAGE},True)
 IPV6_SUPPORT := $(or ${IPV6_SUPPORT}, True)
+OCS_TEST_INTERNAL_BUILD:=$(or ${OCS_TEST_INTERNAL_BUILD},False)
```

I see that in the rest of the file it's used to have spaces around `:=` but it's not the case of those 3 new lines. Probably better to follow the same style in the file.

```suggestion
OCS_TEST_INTERNAL_BUILD := $(or ${OCS_TEST_INTERNAL_BUILD},False)
```